vector optimized configurations (#527)
neo4j/docs-aura: modules/ROOT/pages/managing-instances/instance-details.adoc

The data in your instance can be backed up, exported, and restored using snapshots.
A snapshot is a copy of the data in an instance at a specific point in time.

Neo4j regularly takes snapshots of your instance, and you can also take a snapshot on demand.
These snapshots can be used to restore data to a different Neo4j instance.

> == Vector optimization

Reviewer: ```suggestion
[[aura-vector-optimization]]
== Vector optimization
```
So we can link to the section.

Neo4j regularly takes snapshots of your instance, and you can request a snapshot to be taken on demand.
These snapshots can be used to restore data to a different Neo4j instance.

== Vector optimization

label:AuraDB-Professional[]
label:AuraDB-Business-Critical[]
label:AuraDB-Virtual-Dedicated-Cloud[]

Vector optimization reserves memory for vector indexes, enhancing performance for vector-based operations.
It is available for AuraDB instances with more than 4GB of memory, and is available across all supported cloud providers and regions.

This configuration may impact memory use and therefore may affect performance for other queries.

> You can enable vector optimization during instance creation when selecting *Instance details* with a toggle under *Additional settings*.

Reviewer: ```suggestion
You can enable vector optimization during instance creation using the toggle in *Additional settings*, in the *Instance details*.
```

Author: Thanks for the suggestion - I considered both phrases, and the original phrasing seems a bit clearer as it guides users through a sequential flow, directing them to Instance details before mentioning Additional settings. Let me know if you have any other thoughts!

Author: Update: waiting for the final UX for the setting before finalizing this part

> To enable vector optimization on existing instances, from the instance card, select the more menu *(…​)* then select *Inspect* to see the *Instance details.*

Reviewer: ```suggestion
To enable vector optimization on existing instances, from the instance card, use the more menu *(…​)* to *Inspect*  the instance details.
```

Neo4j regularly takes snapshots of your instance, and you can request a snapshot to be taken on demand.
These snapshots can be used to restore data to a different Neo4j instance.

== Vector optimization

label:AuraDB-Professional[]
label:AuraDB-Business-Critical[]
label:AuraDB-Virtual-Dedicated-Cloud[]

Vector optimization reserves memory for vector indexes, enhancing performance for vector-based operations.
It is available for AuraDB instances with more than 4GB of memory and across all supported cloud providers and regions.

> This configuration re-allocates memory from the graph database to the vector index. 
> If your application needs better vector and graph performance, you should resize to a larger Aura instance.

Reviewer: ```suggestion
This configuration re-allocates memory to the vector index at the expense of non-vector query performance. 
If this has a negative impact on your application, consider resizing your Aura instance.
```
I don't think it is correct to say that the vector index and the graph database are two separate things; the vector index, like any other index, is part of the graph database. Additionally, we need to avoid trying to upsell our users, at least in the docs.

You can enable vector optimization during instance creation by selecting *Instance details* then *Additional settings*. 

To enable vector optimization on existing instances, from the instance card, use the more menu *(…​)* to *Inspect*  the instance details.
You can view the current vector configuration status from the *instance details*.

If you lower the instance size below 4GB, vector optimization is disabled automatically.

If you clone your instance to a new instance, the new instance inherits the vector optimization settings of the original instance.

> But if you clone to an existing instance, it's configuration isn't changed.

Reviewer: ```suggestion
But if you clone to an existing instance, its vector optimization setting isn't changed.
```
